src/Space.cpp

```diff
@@ -157,7 +157,7 @@ Space::Space(Game *game, RefCountedPtr<Galaxy> galaxy, const Json &jsonObj, doub
 Space::~Space()
 {
 	UpdateBodies(); // make sure anything waiting to be removed gets removed before we go and kill everything else
-	for (std::list<Body *>::iterator i = m_bodies.begin(); i != m_bodies.end(); ++i)
+	for (std::vector<Body *>::iterator i = m_bodies.begin(); i != m_bodies.end(); ++i)
 		KillBody(*i);
```

Comment:
```suggestion
	for (Body *body : m_bodies)
		KillBody(body);
```
I'd recommend using a range-for loop here instead; it saves a fair amount of typing and is pretty universally-accepted syntax.

Reply:
done!